ui/app/pages/token/actionable-message/actionable-message.stories.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { text, boolean } from '@storybook/addon-knobs/react'
+import ActionableMessage from '.'
+
+export default {
+  title: 'ActionableMessage',
+}
+
+export const NoAction = () => (
+  <div style={{ height: '200px', width: '200px' }}>
+    <ActionableMessage
+      shown={boolean('Shown', true)}
+      message={text('Message', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')}
+    />
+  </div>
+)
+
+export const OneAction = () => (
+  <div style={{ height: '200px', width: '250px' }}>
+    <ActionableMessage
+      shown={boolean('Shown', true)}
+      message={text('Message', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')}
+      actions={[
+        {
+          label: text('ActionLabel', 'Dismiss'),
+          onClick: action('OneAction Click'),
+        },
+      ]}
+    />
+  </div>
+)
+
+export const TwoActionsWithClassNames = () => (
+  <div style={{ height: '200px', width: '300px' }}>
+    <ActionableMessage
+      shown={boolean('Shown', true)}
+      message={text('Message', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')}
+      actions={[
+        {
+          label: text('First ActionLabel', 'Dismiss'),
+          onClick: action('TwoActionsWithClassNames Click 1'),
+          actionClassName: 'text-transform-uppercase',
```

Comment:
I am deprecating these utility classes in favor of keeping styles inline so that editing/extending/debugging styles is easier. If you could style these a different way I would appreciate it. 

Orthogonally: Are we allowing *too much* flexibility in this API? Would we ever want these actions to be entirely controlled by the developer implementing them? If no, then I'd suggest we apply the styles a different way either using modifiers such as 'uppercase: true' that adds the appropriate class to the action.